src/Glpi/Inventory/Conf.php

```diff
@@ -400,14 +400,14 @@ public function showConfigForm()
 
             echo "<tr>";
             echo "<th colspan='4'>";
-            echo __('Import options');
+            echo __s('Import options');
             echo "</th>";
             echo "</tr>";
 
             echo "<tr class='tab_bg_1'>";
             echo "<td>";
             echo "<label for='import_volume'>";
-            echo \Item_Disk::createTabEntry(\Item_Disk::getTypeName(Session::getPluralNumber()), 0, \Item_Disk::getType());
+            echo \Item_Disk::createTabEntry(htmlspecialchars(\Item_Disk::getTypeName(Session::getPluralNumber())), 0, \Item_Disk::getType());
```

Comment:
IMHO, as `createTabEntry()` is producing the HTML code, it should be responsible of the encoding. We should not expect every caller to handle this.